src/main/java/seedu/tp/ui/Ui.java

```diff
@@ -260,7 +260,7 @@ private void listFlashcardsInPeriod(List<Flashcard> sortedFlashcards, LocalDate
     }
 
     /**
-     * Prints confirmation that study plan has been updated.
+     * Prints confirmation that study plan has been deleted.
```

Comment:
Shouldn't this be kept as 'updated' since it's for the `confirmStudyPlanUpdate` method?

Reply:
Oops, you're right, I meant to change the comment for the method below (deletion). Will update.